orchagent/routeorch.cpp

```diff
@@ -1374,6 +1374,12 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
             if (m_neighOrch->hasNextHop(nexthop))
             {
                 next_hop_id = m_neighOrch->getNextHopId(nexthop);
+                if (m_neighborToResolve.find(nexthop) != m_neighborToResolve.end())
+                {
+                    m_neighOrch->clearResolvedNeighborEntry(nexthop);
+                    m_neighborToResolve.erase(nexthop);
```

Comment:
This clear should be done in the function addNextHop() in neighorch.cpp after successfully creating the NH in the HW.

Reply:
Moved the clear to addNextHop().

Reply:
ok